tests/sentry/tasks/test_unmerge.py

```diff
@@ -273,6 +276,10 @@ def create_message_event(template, parameters):
             (u'sentry:release', u'version', 17),
         ])
 
+        assert features.query(source) == [
+            (source.id, {'message:message:character-shingles': 1.0}),
+        ]
```

Comment:
This is super brittle with regard to changes to feature extraction, I can't think of a better way to do this off of my head that doesn't involve dependency injection and I really don't want to open that can of worms right now if possible.